docs/dev/spatial-data.md

```diff
@@ -0,0 +1,43 @@
+# Spatial data
+
+We store certain geospatial data in our own database and use it for some basic
+geospatial queries. In order for the site to work correctly locally, the data
+needs to be loaded first. The spatial data is stored in our repo using
+[git LFS](https://www.atlassian.com/git/tutorials/git-lfs).
+
+1. Install and enable git LFS.
+
+   - **Linux**: [download instructions](https://docs.github.com/en/repositories/working-with-files/managing-large-files/installing-git-large-file-storage?platform=linux)
+   - **Windows**: [download instructions](https://docs.github.com/en/repositories/working-with-files/managing-large-files/installing-git-large-file-storage?platform=windows)
+   - **macOS**: the easiest way is to use Homebrew and run
+     ```sh
+     brew install git-lfs
+     ```
+     Alternatively, there are manual [download instructions](https://docs.github.com/en/repositories/working-with-files/managing-large-files/installing-git-large-file-storage?platform=mac).
+
+2. Enable git LFS on your local machine by running
+
+   ```sh
+   git lfs install
+   ```
+
+3. Load the spatial data into your local database by running
```

Comment:
Maybe for the future not-usses (or us, when we've long forgotten), we can reiterate that you need to zap the application locally here and also `git-lfs pull` if needed. Or at least just to say, make sure the containers are running locally before doing this step.

Reply:
Went ahead and updated the doc, but also added spatial loading into the `make zap` command.